common/app/views/fragments/items/facia_cards/sliceSlot.scala.html

```diff
@@ -0,0 +1,20 @@
+@(
```

Comment:
There is a lot of redundancy between this fragment and `gallerySlot`, and in the future we might have a similar `liveblogSlot`. Only the class names change (I might do something about that in the future) if I'm not mistaken, I guess we could abstract that some more.